cloudevents/languages/zh-CN/primer.md

```diff
@@ -665,7 +666,7 @@ CloudEvents 将为任何事件使用者（例如serverless平台）提供一种
 }
 ```
 
-#### AWS - CloudWatch Events
+#### AWS - CloudWatch Events/亚马逊-CloudWatch 事件
```

Comment:
very minor thing, but can we put spaces before/after the "-" here and on line 692 ?

Reply:
sure. I will add them